packages/ove-service-auth/README.md

```diff
@@ -0,0 +1,191 @@
+# OVE Base Proxy
+
+This is a generic base proxy that supports various configurations, provided either via environment variables or 
+configuration files.
+
+## Configurations
+
+### Request Proxy
+
+The request proxy supports two configurations: a global or service-based proxy. The service proxy supports basic
+load balancing.
+
+The proxy mechanism can be enabled by setting the **SERVICE_PROXY_ENABLED** environment variable, the default is **True**.
+
+In proxy mode the service maps all requests based on a path matching strategy:
```

Comment:
```suggestion
The service maps all requests based on a path matching strategy:
```

What does "in proxy mode" mean - doesn't the proxy always act as a proxy?

Reply:
Not necessarily, it can be just an auth service

Reply:
Ok. I was slightly confused by what the README was saying about how these options behaved. 

After reading the [relevant code](https://github.com/ove/ove-services/blob/f2cfb6a95315de9ab30ed133c797c971a4ec0ede/packages/ove-service-auth/heimdall/__init__.py#L63), I suggest explicitly saying that:

- if `SERVICE_GLOBAL_PROXY_URL` (copied to `proxy_forward`) is set, then the service works in **global proxy mode**

- otherwise, if `SERVICE_PROXY_ENABLED` (copied to `proxy_enabled`) is `True`, then the service works in **service proxy mode**

- otherwise it does not act as a proxy